app/js/api-service.js

```diff
@@ -253,16 +233,32 @@ angular.module('cttvServices')
 
 
         /**
-        * TODO:
-        * this needs to be consolidated with the getAssociation() method as they are the exact same thing...
+        *
         */
         cttvAPI.getAssociations = function(queryObject){
+            queryObject[ cttvAPI.facets.SCORE_MIN ] = queryObject[ cttvAPI.facets.SCORE_MIN ] || [0.0] ;
             $log.log("cttvAPI.getAssociations()");
             $log.log(queryObject);
-            // queryObject[ cttvAPI.facets.SCORE_STR ] = queryObject[ cttvAPI.facets.SCORE_STR ] || [1] ; // No need for stringency for now
-            queryObject[ cttvAPI.facets.SCORE_MIN ] = queryObject[ cttvAPI.facets.SCORE_MIN ] || [0.0] ;
 
+            queryObject[ cttvAPI.facets.SCORE_MIN ] = 0.0;
+            return callAPI({
+                operation : cttvAPI.API_ASSOCIATION_URL,
+                params : queryObject
+            });
+        };
+
+        cttvAPI.getAssociationsPost = function(queryObject){
```

Comment:
I have made a pull request against master to avoid having this method. Don't worry for now, I'll merge once it is revised (Luca is doing it)